images/solr/00-solr-datadir.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+set -ex
+
+# old volume looks like this
+# $corename/data/-files-
+# new volume looks like this
+# $corename/-files-
+
+for datadir in `ls -d /var/solr/*` ; do
```

Comment:
let's use `$()` instead of backticks, as backticks are not suggested by bash: https://stackoverflow.com/a/33301370

Reply:
updated: 63a7061d8bc9792c21f59e300dda3803f9fab822

Reply:
fixed in 63a7061d8bc9792c